include/boost/url/impl/pct_encoding.ipp

```diff
@@ -133,6 +132,89 @@ pct_decode_unchecked(
     return dest - dest0;
 }
 
+BOOST_URL_DECL
```

Comment:
I think, you don't put `BOOST_URL_DECL` on the definition?

Reply:
Ooops... bad copying and pasting.